services/ui-src/src/components/users/UserEdit.js

```diff
@@ -19,12 +22,60 @@ const UserEdit = ({ stateList }) => {
 
   // Set up local state
   const [user, setUser] = useState();
-  /* eslint-disable no-unused-vars */
-  const [states, setStates] = useState();
   const [role, setRole] = useState();
   const [isActive, setIsActive] = useState();
+
+  const [states, setStates] = useState(stateList);
+  const [selectedStates, setSelectedStates] = useState([]);
   /* eslint-disable no-unused-vars */
-  const [statesToSend, setStatesToSend] = useState("null");
+  const [statesToSend, setStatesToSend] = useState([]);
+
+  // Get User data
+  const loadUserData = async () => {
+    // Retrive user data from datastore
+    const getUserData = { userId: id };
+    const data = await getUser(getUserData);
+    setUser(data);
+    setRole(data.role);
+    setStates(stateList);
+
+    // Sort states alphabetically
+    const theStates = data.states.sort();
+
+    // Set states to array of objects
+    if (data.role !== "state") {
+      setSelectedStates(
+        theStates.map(e => {
+          let stateName;
+          // Get full state name from redux
+          for (const state in stateList) {
+            if (stateList[state].value === e) {
+              stateName = stateList[state].label;
+            }
+          }
+          return { label: stateName, value: e };
```

Comment:
planning on refactoring this block, due to the fact that they share very similar functionality